src/panels/lovelace/cards/hui-picture-entity-card.js

```diff
@@ -24,6 +24,7 @@ class HuiPictureEntityCard extends EventsMixin(LocalizeMixin(PolymerElement)) {
       <style>
         ha-card {
           min-height: 75px;
+          max-height: 100%;
```

Comment:
I really am not sure if this is the right solution. would we want the floor plan to always fit in the screen? It might get very small on a tablet in landscape?